include/pybind11/pybind11.h

```diff
@@ -452,6 +452,10 @@ class cpp_function : public function {
 
     /// When a cpp_function is GCed, release any memory allocated by pybind11
     static void destruct(detail::function_record *rec) {
+        #if !defined(PYPY_VERSION) && PY_MAJOR_VERSION == 3 && PY_MINOR_VERSION == 9
+            static bool is_zero = Py_GetVersion()[4] != '0';
```

Comment:
The variable is called `is_zero`, but the value it holds is the opposite.

Reply:
Whoops. And then there's Boris, actually bringing up issues with the code...

Reply:
😳